clang/test/SemaSYCL/unsupported_math.cpp

```diff
@@ -30,14 +26,6 @@ int main() {
     acc[0] += (int)log(1.0);                    // expected-no-error
     acc[0] += (int)__builtin_logf(1.0f);        // expected-no-error
     acc[0] += (int)__builtin_log(1.0);          // expected-no-error
-    acc[0] += (int)ceilf(1.0f);                 // expected-error{{builtin is not supported on this target}}
```

Comment:
```suggestion
    acc[0] += (int)ceilf(1.0f);                 // expected-no-error
```

Reply:
I think this is overkill as we could end up adding (expected-no-error) for all the supported math library functions.
I agree that my error in the compiler could have been caught by this though.

Reply:
:)